src/components/form-helpers/form-helpers.js

```diff
@@ -343,7 +351,11 @@ export const InputFieldItem = ({
       name={fieldName}
       id={fieldName}
       placeholder={placeholder}
-      validate={createRequiredValidation(required)}
+      validate={combineValidations(
+        createRequiredValidation(required),
+        minLength && createMinLengthValidation(minLength),
+        withNameValidation && validateName,
```

Comment:
Did you test this manually? I mean, for example, what happens when _minLength_ is _falsy_, and _withNameValidation_ is _true_?

Reply:
Ready fixed